### DISCUSSION TITLE
A new, **un**official FANDOM account for ProdigyMathGameHacking.

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#780] [(A) Anything]
It's called ProdigyMathGameHacking Official.
I will be the primary one working on this FANDOM account.
Thanks,
hellonearth311.

**Comment by ghost** [comment]
it aint official if you made it

**Comment by hellonearth311** [comment]
bruh how?

**Reply by NX721** [reply]
Let the devs of PMGH make it

**Comment by hellonearth311** [comment]
ill rename it but bruhh

**Comment by BoredFishRE** [comment]
bro excuse me? We NEVER gave you permission to do that. Just because you contribute doesn't mean you can create official accounts for us.

**Comment by NX721** [comment]
@hellonearth311

**Comment by ghost** [comment]
That makes no sense

its like I make an official account for bill gates because I know his name @hellonearth311

**Comment by LeoBadeaux** [comment]
It's fine, you can make your own little fandom for us, but you can't call it official.
